apps/greencheck/importers/importer_csv.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import logging
+import pandas as pd
+import ipdb
+import re
+import ipaddress
+
+from apps.greencheck.importers.importer_interface import BaseImporter, Importer
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class CsvImporter(BaseImporter):
+    # def __init__(cls):
+    # cls.hosting_provider_id = settings.Csv_PROVIDER_ID
+
+    def fetch_data_from_source(cls):
+        # TODO: fetch data from website dropping
+        # For now: expect that a csv is saved somewhere and fetch in the following way:
+        raw_data = pd.read_csv(
+            "apps/greencheck/fixtures/test_dataset_csv.csv", header=None
```

Comment:
Can you please use the pathlib library when referring to files at paths instead of strings?

https://treyhunner.com/2019/01/no-really-pathlib-is-great/